openemrs/app/models/user.rb

```diff
@@ -0,0 +1,29 @@
+class User < ApplicationRecord
+  include GenerateUid
+  REGEX = /\A([\w+\-].?)+@[a-z\d\-]+(\.[a-z]+)*\.[a-z]+\z/i
```

Comment:
There should be some gem which abstracts this out nicely. It's fine for now, since this regex — from me just eyeballing it — looks like it covers the cases we want to worry about for a V1. As things get more complex, we may need to abstract this into a gem, or find a gem which already does that for us.

Reply:
Found these gems for email validation - https://github.com/rubygarage/truemail and https://github.com/K-and-R/email_validator

Reply:
I was just about to tell you about Truemail. I like what they've done with the gen. 

Let's bring Truemail in.

Reply:
I'll add a story to the board for this